app/iht/config/ApplicationGlobal.scala

```diff
@@ -36,13 +40,37 @@ object ApplicationGlobal extends DefaultFrontendGlobal with RunMode {
   override val frontendAuditFilter = IhtAuditFilter
   implicit val formPartialRetriever: FormPartialRetriever = IhtFormPartialRetriever
 
+  private implicit def rhToRequest(rh: RequestHeader): Request[_] = Request(rh, "")
+
   override def onStart(app: Application) {
     super.onStart(app)
     ApplicationCrypto.verifyConfiguration()
   }
 
+  override def internalServerErrorTemplate(implicit request: Request[_]): Html = {
```

Comment:
Could use double checking the super.resolveError doesn't call this or rename it to something else